agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java

```diff
@@ -616,6 +616,12 @@ private static String nullAwareConcat(String str1, String str2, String separator
 
     private void exportEvents(SpanData span, Double samplingPercentage) {
         for (EventData event : span.getEvents()) {
+            boolean lettuce51 =
+                    span.getInstrumentationLibraryInfo().getName().equals("io.opentelemetry.javaagent.lettuce-5.1");
```

Comment:
We only suppress lettuce-5.1 and allow other versions if available? What about lettuce-5.0 and 4.0?